### PR TITLE
config-linux: Fix example of cpu quota

### PR DESCRIPTION
**Pull request by hqhq**
Quota should be smaller than period length.

Signed-off-by: Qiang Huang <h.huangqiang@huawei.com>

### CODE REVIEW

**Review by giuseppe — APPROVED** [review]
LGTM

File: config-linux.md [hunk 1]
```diff
@@ -371,8 +371,8 @@ The following parameters can be specified to set up the controller:
 ```json
 "cpu": {
     "shares": 1024,
-    "quota": 1000000,
-    "period": 500000,
+    "quota": 500000,
```
Comment (kailun-qin) [inline config-linux.md:374]: Why quota needs to be smaller than period? The config seems valid on a multi-CPU machine.

File: config-linux.md [hunk 2]
```diff
@@ -371,8 +371,8 @@ The following parameters can be specified to set up the controller:
 ```json
```
Comment (kailun-qin) [inline config-linux.md:371]: Do we need to change values in `schema/test/config/good/spec-example.json` accordingly in this case?

File: config-linux.md (hunk 1 above)
Comment (tianon) [reply config-linux.md:374]: Indeed -- this change would effectively mean a single container could never use more than one core's worth of CPU time.

When I use Docker's `--cpus` shorthand for setting these options, here's what `/sys/fs/cgroup/cpu.max` looks like:

```console
$ docker run --rm --cpus 2 bash cat /sys/fs/cgroup/cpu.max
200000 100000
$ docker run --rm --cpus 0.5 bash cat /sys/fs/cgroup/cpu.max
50000 100000
```

(Both very legal and sane configurations.)

**Review by tianon — CHANGES_REQUESTED** [review]
(for clarity, adding a "review" as an explicit NACK -- see discussion above)

File: config-linux.md (hunk 1 above)
Comment (hqhq) [reply config-linux.md:374]: Never mind, not saying it was a mistake, just figured quota less than period would be more understandable, extra description might needed to explain the multi cpu case (like examples in: https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt) which I thought can be avoided.

Thanks for your comments.

**Pull request closed without merging**